tests/x11regressions/firefox/firefox_java.pm

```diff
@@ -16,45 +16,44 @@ use base "x11regressiontest";
 use testapi;
 
 sub java_testing {
+    my ($self) = @_;
+
     send_key "ctrl-t";
     send_key "alt-d";
     type_string "http://www.java.com/en/download/installed.jsp?detect=jre\n";
 
     wait_still_screen 3;
+    check_screen('oracle-cookies-handling');
```

Comment:
Please do not use `check_screen` with default timeout and definitely not without checking the return value.